File: app/src/main/java/ru/ulstu_team/ulstuschedule/util/TimeStringUtils.java
package ru.ulstu_team.ulstuschedule.util;

public class TimeStringUtils {

    private static String[] beginHours =
            new String[]{"8", "9", "11", "13", "14", "16", "18", "19"};
    private static String[] beginMinutes =
            new String[]{"00", "40", "30", "10", "50", "30", "10", "50"};
    private static String[] endHours =
            new String[] {"9", "11", "13", "14", "16", "18", "19", "21"};
    private static String[] endMinutes =
            new String[] {"30", "10", "00", "40", "20", "00", "40", "20"};

    public static String beginHour(int lessonNumber) {
        return beginHours[lessonNumber - 1];
    }

    public static String beginMinutes(int lessonNumber) {
        return beginMinutes[lessonNumber - 1];
    }

    public static String endHour(int lessonNumber) {
        return endHours[lessonNumber - 1];
    }

    public static String endMinutes(int lessonNumber) {
        return endMinutes[lessonNumber - 1];
    }
}

File: app/src/main/java/ru/ulstu_team/ulstuschedule/util/TimeStringUtils.kt
package ru.ulstu_team.ulstuschedule.util

object TimeStringUtils {

    private val beginHours = arrayOf("8", "9", "11", "13", "14", "16", "18", "19")
    private val beginMinutes = arrayOf("00", "40", "30", "10", "50", "30", "10", "50")
    private val endHours = arrayOf("9", "11", "13", "14", "16", "18", "19", "21")
    private val endMinutes = arrayOf("30", "10", "00", "40", "20", "00", "40", "20")

    fun beginHour(lessonNumber: Int) = beginHours[lessonNumber - 1]

    fun beginMinutes(lessonNumber: Int) = beginMinutes[lessonNumber - 1]

    fun endHour(lessonNumber: Int) = endHours[lessonNumber - 1]

    fun endMinutes(lessonNumber: Int) = endMinutes[lessonNumber - 1]
}
